ipa_room_exploration/ros/include/ipa_room_exploration/room_exploration_action_server.h

```diff
@@ -91,10 +91,13 @@
 #include <geometry_msgs/PoseStamped.h>
 #include <geometry_msgs/Polygon.h>
 #include <geometry_msgs/Point32.h>
+#include <nav_msgs/Path.h>
 #include <nav_msgs/OccupancyGrid.h>
 #include <sensor_msgs/Image.h>
 #include <sensor_msgs/image_encodings.h>
 // specific from this package
+#include <ipa_navigation_utils/utils.h>
```

Comment:
I think this package is public? We have this software public and free for academic use. So we should either add a compile flag for using ipa_navigation or you need to make the ipa_navigation_utils public.